pkg/api/controller.go

```diff
@@ -369,14 +370,15 @@ func (c *Controller) StartBackgroundTasks(reloadCtx context.Context) {
 
 			// Enable extensions if extension config is provided for subImageStore
 			if c.Config != nil && c.Config.Extensions != nil {
-				ext.EnableExtensions(c.Config, c.Log, storageConfig.RootDirectory)
+				ext.EnableMetricsExtension(c.Config, c.Log, storageConfig.RootDirectory)
```

Comment:
Is it possible to make the code outside pkg/extensions unaware of what extensions exist/are enabled?
Maybe as a separate PR.